packages/hash/shared/src/environment.ts

```diff
@@ -1,4 +1,19 @@
-export const apiOrigin =
-  process.env.NEXT_PUBLIC_API_ORIGIN ?? "http://localhost:5001";
+export const apiOrigin = process.env.NEXT_PUBLIC_API_ORIGIN
+  ? process.env.NEXT_PUBLIC_API_ORIGIN
+  : process.env.API_ORIGIN
+  ? process.env.API_ORIGIN
+  : "http://localhost:5001";
```

Comment:
I was gonna do this, but realized the "default chain" we have underneath requires a ternary and wanted to keep them consistent. But I'm fine with this change :)

Reply:
We can replace `??` with `||` to avoid getting `""` here. This can happen when someone 'reset' the var via `FOO=`

Reply:
@kachkaev I assume because ?? wouldn't go to the right-hand side if the value is empty?

Reply:
We have a link that doesn't allow this:
```
Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator.eslint[@typescript-eslint/prefer-nullish-coalescing](https://typescript-eslint.io/rules/prefer-nullish-coalescing)
```

Reply:
Addressed in a9bd08483ae1e9522d65af95062bd515ba4a2f39